src/pages/Products/Products.js

```diff
@@ -25,11 +30,15 @@ export function Products() {
 
   useEffect(() => {
     loadPosts();
-  }, []);
+  }, [currentPage]);
+
+  const onPageChange = (pageNumber) => {
+    setCurrentPage(pageNumber);
```

Comment:
add pageNumber to query string in order to fetch corresponding bunch of items on page reload